apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -1447,8 +1449,106 @@ describe('LIVECHAT - rooms', function () {
+			await updatePermission('close-others-livechat-room', []);
+			const visitor = await createVisitor();
+			const { _id } = await createLivechatRoom(visitor.token);
+			await request.post(api('livechat/room.closeByUser')).set(credentials).send({ rid: _id }).expect(400);
+		});
+		it('should close room if user has permission', async () => {
+			await updatePermission('close-others-livechat-room', ['admin']);
+			const visitor = await createVisitor();
+			const { _id } = await createLivechatRoom(visitor.token);
+			await request.post(api('livechat/room.closeByUser')).set(credentials).send({ rid: _id }).expect(200);
+		});
+	});
+
+	(IS_EE ? describe : describe.skip)('omnichannel/:rid/request-transcript', () => {
+		before(async () => {
+			await updateSetting('Livechat_Routing_Method', 'Manual_Selection');
+			// Wait for one sec to be sure routing stops
+			await sleep(1000);
+		});
+
+		it('should fail if user is not logged in', async () => {
+			await request.post(api('omnichannel/rid/request-transcript')).expect(401);
+		});
+		it('should fail if :rid doesnt exists', async () => {
+			await request.post(api('omnichannel/rid/request-transcript')).set(credentials).expect(400);
+		});
+		it('should fail if user doesnt have request-pdf-transcript permission', async () => {
+			await updatePermission('request-pdf-transcript', []);
+			const visitor = await createVisitor();
+			const { _id } = await createLivechatRoom(visitor.token);
+			await request
+				.post(api(`omnichannel/${_id}/request-transcript`))
+				.set(credentials)
+				.expect(403);
+		});
+		// Increasing a bit the timeout since service calls + other calls are a bit slow on pipe
+		it('should fail if room is not closed', async () => {
+			await updatePermission('request-pdf-transcript', ['admin', 'livechat-agent', 'livechat-manager']);
+			const visitor = await createVisitor();
+			const { _id } = await createLivechatRoom(visitor.token);
+			await request
+				.post(api(`omnichannel/${_id}/request-transcript`))
+				.set(credentials)
+				.expect(400);
+		}).timeout(15000);
+		it('should fail if no one is serving the room', async () => {
+			const visitor = await createVisitor();
+			const { _id } = await createLivechatRoom(visitor.token);
+			await closeRoom(_id);
+			await request
+				.post(api(`omnichannel/${_id}/request-transcript`))
+				.set(credentials)
+				.expect(400);
+		}).timeout(15000);
+		let roomId: string;
+		it('should request a pdf transcript when all conditions are met', async () => {
+			await createAgent();
+			const visitor = await createVisitor();
+			const { _id } = await createLivechatRoom(visitor.token);
+			const inq = await fetchInquiry(_id);
+			roomId = _id;
+			await takeInquiry(inq._id);
+			await closeRoom(_id);
+
+			await request
+				.post(api(`omnichannel/${_id}/request-transcript`))
+				.set(credentials)
+				.expect(200);
+		}).timeout(15000);
```

Comment:
why such huge timeouts? looking the tests that executed on CI, you could keep using the default one

Reply:
Ah, this was for the other bug we were investigating. I forgot to reset this timeout